subsys/bluetooth/host/Kconfig

```diff
@@ -436,12 +440,13 @@ config BT_PERIPHERAL_PREF_SLAVE_LATENCY
 config BT_PERIPHERAL_PREF_TIMEOUT
 	int "Peripheral preferred supervision timeout in 10ms units"
 	default 42
-	range 10 3200
+	range 0 3200
 	help
 	  It is up to user to provide valid timeout which pass required minimum
 	  value: in milliseconds it shall be larger than
 	  "(1+ Conn_Latency) * Conn_Interval_Max * 2"
 	  where Conn_Interval_Max is given in milliseconds.
+	  Value 0 disables this configuration.
```

Comment:
I don't think this is accurate. The spec says "no specific value" which is different from "disabled"

Reply:
I assume there is no difference in practice since that means the value is not going to be used, anyway I will use the exact working of the spec.

Reply:
@Vudentz I was talking here what the values mean to the GATT client that reads them, which is also what the GAP spec wording has in mind, I think. In that sense "use whatever value you want" is more intuitive than "this is disabled". If you look at this from a Kconfig description, then I can understand "disabled" a bit better since it means we're not requesting any specific value. Either way, I'd just stick to the spec wording to avoid confusing people.